mangum/handlers/aws_alb.py

```diff
@@ -71,22 +86,37 @@ def encode_query_string(self) -> bytes:
 
         return urllib.parse.urlencode(query).encode()
 
+    def transform_headers(self) -> List[Tuple[bytes, bytes]]:
+        """Convert headers to a list of two-tuples per ASGI spec.
+
+        Only one of `multiValueHeaders` or `headers` should be defined in the
+        trigger event. However, we act as though they both might exist and pull
+        headers out of both.
+        """
+        headers = []
+        for k, v in self.trigger_event.get("multiValueHeaders", {}).items():
```

Comment:
How is the behavior here different than what occurs in the original implementation [here](https://github.com/jordaneremieff/mangum/blob/ded6a55a3ca65c26bf9228ed8d78c2a72bb9c82b/mangum/adapter.py#L94)? 

Would it be possible to revert to using a similar approach from that version, or is there an issue with how that version works?

Reply:
The old version you linked to forces you to do janky stuff like [splitting the string by comma](https://github.com/jordaneremieff/mangum/blob/ded6a55a3ca65c26bf9228ed8d78c2a72bb9c82b/mangum/adapter.py#L116).

And from looking at it just now it doesn't seem like it's consistent with the [ASGI spec](https://github.com/jordaneremieff/mangum/blob/ded6a55a3ca65c26bf9228ed8d78c2a72bb9c82b/mangum/adapter.py#L116) - but I could be wrong, I'm not expert on it. [This](https://github.com/jordaneremieff/mangum/blob/ded6a55a3ca65c26bf9228ed8d78c2a72bb9c82b/mangum/adapter.py#L94-L96) combined with [this](https://github.com/jordaneremieff/mangum/blob/ded6a55a3ca65c26bf9228ed8d78c2a72bb9c82b/mangum/adapter.py#L150) means that it's combining all header values into a comma separated string (bytes) rather than having them as independent name/value pairs.